src/Main.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Scanner workTime = new Scanner(System.in);
+        int hour = workTime.nextInt();
+
+        if (hour == 12) {
+            System.out.println("It's time for lunch");
+        }
```

Comment:
in java ist es üblich, dass das `else if` bzw `else` in der gleichen zeile wie die schließenden und öffnenden klammern sind - und an die java konventionen sollten wir uns halten 🙏 

also bitte `} else if (...) {`